asrock/spc741d8/README.md

```diff
@@ -0,0 +1,25 @@
+# ASRock Rack SPC blobs SPC741D8-2L2T
+
+The directory contains blobs to produce a full Dasharo firmware image
+compatible with ASRock Rack SPC741D8-2L2T.
```

Comment:
Add `/BCM` suffix to the model here too? https://www.asrockrack.com/general/productdetail.asp?Model=SPC741D8-2L2T/BCM#Specifications has it.

Reply:
Fixed in https://github.com/Dasharo/dasharo-blobs/pull/44/commits/3d57c0915eba602fb50089281d930e971060b04d